Program.cs

```diff
@@ -5,11 +5,91 @@ namespace Calculator
   public static class Program
   {
     private static void Main(string[] args)
+    {
+      PrintWelcomeMessage();
+
+      while (true)
+      {
+        PerformOneCalculation();
+      }
+    }
+
+    private static void PrintWelcomeMessage()
     {
       Console.WriteLine("Welcome to the calculator!");
       Console.WriteLine("==========================");
+    }
+
+    private static void PerformOneCalculation()
+    {
+      var op = AskForOperator();
+      var n = AskForNumberArray(op);
+      var a = CalculateAnswer(op, n);
```

Comment:
More descriptive names for the variables would help readability i.e. var operator.